ntbs-integration-tests/NotificationPages/ClinicalDetailsPageTests.cs

```diff
@@ -7,15 +7,15 @@
 using ntbs_service.Models.Validations;
 using Xunit;
 
-namespace ntbs_integration_tests
+namespace ntbs_integration_tests.NotificationPages
 {
     public class ClinicalDetailsPageTests : TestRunnerBase
     {
         protected override string PageRoute

```

Comment:
Also not part of this changeset but could be converted to expression-bodied property (`=> Routes.ClinicalDetails`)

Reply:
yep, will do!